docs/content/https/tls.md

```diff
@@ -159,6 +159,75 @@ data:
 
 If no default certificate is provided, Traefik generates and uses a self-signed certificate.
 
+### ACME Default Certificate
+
+You can configure Traefik to use an ACME provider (like Let's Encrypt) to generate the default certificate.
+The configuration to resolve the default certificate should be defined in a TLS store:
+
+!!! important "Precedence with the `defaultCertificate` option"
+
+    The `defaultCertificate` definition takes precedence on the ACME default certificate configuration.
+
+```yaml tab="File (YAML)"
+# Dynamic configuration
+
+tls:
+  stores:
+    default:
+      defaultCertResolver: myresolver
+      defaultCertDomain:
+        main: foobar
+        sans:
```

Comment:
The definitions for `main` and `sans` aren't explained in this document as far as I can see – they probably should, or they should be linked somewhere that does explain them?